apps/coscrad-frontend/src/components/home/home.tsx

```diff
@@ -25,8 +34,11 @@ export const Home = (): JSX.Element => {
     });
 
     return (
-        <div>
-            <img className="home-image" src={siteHomeImageUrl} alt="Home" /> {siteDescription}
-        </div>
+        <>
+            <div>
+                <ImageInContent image={image} alignment="left" displayWidth="350px" />
```

Comment:
The naming is a bit awkward, but I like the idea. Not sure I like the layout props, though. We need to separate presentation from data flow (image url). Also how does this image differ from our photograph view model? I was a bit confused by this in-line model here. But I definitely like your idea of having this Image component.

Reply:
`ImageInContent` is intended to be used to float an image in a page of content in the way we would in our CMS.  For this reason the alignment must be an option to specify when the image is used.  As we talked about, there are many changes to come as we eventually move towards a CMS system.  It makes complete sense to use the Photograph model from the Web of Knowledge to place images in front matter.  Lots to do here...